Update transport docs and settings for changes (#36786)
elastic/elasticsearch: docs/reference/modules/remote-clusters.asciidoc

  are sent according to the global `transport.ping_schedule` setting, which
  defaults to ``-1` meaning that pings are not sent.

`cluster.remote.${cluster_alias}.transport.compress`::

>   Per cluster boolean setting that allows compression to be configured for

Reviewer: ```suggestion
  Per cluster boolean setting that enables you to configure compression for
```

>   requests to a specific remote cluster. This setting only impacts requests

Reviewer: ```suggestion
  requests to a specific remote cluster. This setting impacts only requests
```

>   sent to the remote cluster. Elasticsearch will compress a response if the

Reviewer: ```suggestion
  sent to the remote cluster. If the
```

>   inbound request was compressed. If unset, the global `transport.compress`

Reviewer: ```suggestion
  inbound request is compressed, Elasticsearch compresses the response. If unset, the global `transport.compress`
```
